gettsim/parameters/arbeitsl_geld.yaml

```
@@ -188,3 +164,30 @@ stundengrenze:
   2012-04-01:
     scalar: 15
     reference: Art. 2 G. v. 20.12.2011 BGBl I S. 2854
+
+anspruchsdauer:
+  name:
+    de: Anspruchsdauer für ALG 1
+    en: lenght of ALG 1 eligibility
+  description:
+    de: >-
+      Dauer des Anspruchs auf ALG 1 richtet sich nach Alter und wie viele Monate
+      eine Person versicherungsplichtig in den letzten 5 Jahren war.
+    en: >-
+      The lenght of ALG 1 eligibility depends on age and on how many months a
+      person was subject to compulsory insurance
+  reference: § 147 Abs. 2 SGB III
+  1997-03-24:
+    nach_alter:
+      0: 12
+      50: 15
+      55: 18
+      58: 24
+    nach_versich_pfl:
```

```suggestion
    nach_versicherungspflichtige_monate:
```
No character limit on parameters. Can be more verbose.